apps/serdes/Deserializer.h

```diff
@@ -0,0 +1,32 @@
+#ifndef HALIDE_DESERIALIZER_H
+#define HALIDE_DESERIALIZER_H
+
+#include <string>
+#include <vector>
+
+#include "halide_ir_generated.h"
+#include <Halide.h>
+using namespace Halide;
```

Comment:
It is *never* ok to do `using namespace Whatever;` in a .h file; it means that everything that includes this file will get this namespace opened, which can make for awful errors sometimes. Spell out all namespaces in header files.

(It's occasionally ok to do `using namespace` in `.cpp` files -- we do this frequently in tests -- but aside from tests, avoid this entirely; either do `using Halide::Internal::Whatever;` or, in some cases, `using alias = Really::Long::Namespace::Sequence;`)

Reply:
Thanks for pointing this out!